### PR TITLE
Fix create document

### PR DESCRIPTION
**Pull request by LucasXu0**
### Feature Preview

---

#### PR Checklist

- [x] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/rust-lib/flowy-folder2/src/manager.rs [hunk 1]
```diff
@@ -323,6 +323,14 @@ impl Folder2Manager {
           folder.set_current_view("");
         }
       }
+
+      // notify the parent view that the view is moved to trash
+      send_notification(view_id, FolderNotification::DidMoveViewToTrash)
+        .payload(DeletedViewPB {
+          view_id: view_id.to_string(),
+          index: None,
```
Comment (LucasXu0) [inline frontend/rust-lib/flowy-folder2/src/manager.rs:331]: @appflowy what is the index used for?

Comment (appflowy) [reply frontend/rust-lib/flowy-folder2/src/manager.rs:331]: Index of the view

**Comment by LucasXu0** [comment]
close #2684

**Comment by LucasXu0** [comment]
close #2662

**Comment by LucasXu0** [comment]
close #2658

**Pull request merged**